src/components/common/TmBalance.vue

```diff
@@ -124,6 +158,7 @@ export default {
     ModalTutorial
   },
   filters: {
+    removeUFromMicroDenom,
```

Comment:
this already tells me something is wrong. the UI should not have to handle micro denoms in views

Reply:
I thought I had deleted this one... but good that is signaling something wrong 👍